content/common/contributing-docs.md

```diff
@@ -125,7 +134,60 @@ The no copy annotation (`<!-- @nocpy -->`) precedes a code block where no copy a
 Some generic code not intended for copying/pasting
 ```
 
-**Table of Contents (ToC)**
+Multi-tab code blocks [have recently been implemented](https://github.com/fermyon/developer/pull/239). Examples can be seen in the [Spin](https://developer.fermyon.com/spin/install#installing-spin) installer documentation](https://developer.fermyon.com/spin/install#installing-spin) and [Spin Key/Value documentation](https://developer.fermyon.com/spin/kv-store#the-spin-toml-file). The above examples demonstrate how tabs can either represent platforms i.e. `Windows`, `Linux` and `macOS` or represent specific programming languages i.e. `Rust`, `JavaScript` and `Golang` etc. Here is a brief example of how to implement multi-tab code blocks when writing technical documentation for this site, using markdown.
+
+The first step to implementing multi-tab code blocks is placing the `enable_shortcodes = true` configuration at the start of the `.md` file. Specifically, in the `.md` file's frontmatter.
+
+The markup to create tabs in markdown is as follows 
+
+```
+{{ tabs "Os" }}
+
+{{ startTab "Windows"}}
+
+To list files on windows use `dir`
+
+<!-- @selectiveCpy -->
+
+\`\`\`bash
+$ dir hello_fermyon
+\`\`\`
+and script in windows have the extension `.bat`
+
+<!-- @nocpy -->
+
+\`\`\`bash
+hello.bat
+test.bat
+\`\`\`
+
+{{ blockEnd }}
+
+{{ startTab "Linux"}}
+
+To list files on linux use `ls`
+
+<!-- @selectiveCpy -->
+
+\`\`\`bash
+$ ls
+\`\`\`
+
+and script in windows have the extension `.sh`
```

Comment:
script in `linux`

Reply:
Fixed in new push, thanks again; nice catch.